marimo/_plugins/ui/_impl/input.py

```diff
@@ -1132,9 +1133,11 @@ def __init__(
         label: str = "click here",
         on_change: Optional[Callable[[Any], None]] = None,
         full_width: bool = False,
+        keyboard_shortcut: Optional[str] = None,
     ) -> None:
         self._on_click = (lambda _: value) if on_click is None else on_click
         self._initial_value = value
+        tooltip = keyboard_shortcut if tooltip is None else tooltip
```

Comment:
you may want to push this check into the frontend, so you know to wrap the shortcut in `<KeyboardHotkeys`. since you will need to do `const tooltipContent = keyboardShortcut && !tooltip ? <KeyboardHotkeys keyboardShortcut={keyboardShortcut}/> : tooltip`